Source/WTF/wtf/BitVector.h

```diff
@@ -140,13 +138,17 @@ class BitVector final {
     bool quickGet(size_t bit) const
     {
         ASSERT_WITH_SECURITY_IMPLICATION(bit < size());
+    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
         return !!(bits()[bit / bitsInPointer()] & (static_cast<uintptr_t>(1) << (bit & (bitsInPointer() - 1))));
+    WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
```

Comment:
Surprised by the indentation. I’d expect either fully left aligned, or nested inside the braces, perhaps with the code indented. Haven’t seen this style before.

Reply:
I'll double check I thought this was the style we were using. It's also possible `WTF_ALLOW_UNSAFE_BUFFER_USAGE` macros are currently inconsistent with other macros in this regard.

Reply:
Oh never mind, it's hard reading a diff. This is meant to be fully left-aligned, always. I'll fix.